android/assets/jsons/Civ V - Vanilla/UnitPromotions.json

```diff
@@ -538,5 +538,9 @@
 	{
 		"name": "Pictish Courage", // only for Pictish Warrior and subsequent upgrades
 		"uniques": ["No movement cost to pillage", "+[20]% Strength in [Foreign Land]"]
+	},
+	{
+		"name": "Home Sweet Home", // only for Mehal Sefari and subsequent upgrades
+		"uniques": ["[+30]% Strength decreasing with distance from the capital"]
```

Comment:
Is it reasonably doable to make this something like "[+30]% Strength <decreasing over [3] tiles from the capital>" to allow extra moddability? That would also increase how easily it can be combined with other strength conditionals: `[+30]% Strength <when attacking> <decreasing [...]>` sounds better than `[+30]% Strength <decreasing [...]> <when attacking>` in my opinion, and this might change in other languages

Reply:
Conditionals at the moment are booleans testing something. I didn't want to change that system too fundamentally.

Reply:
Good point, maybe I'm too overzealous converting everything to conditionals and should actually think before propsing :)

Reply:
Thinking of Uniques as potentially json-programmable by recombination is good, but we're still a long way from what e.g. the aurora toolset allowed in the neverwinter series...